### ISSUE TITLE
AberrantExpression/pipeline/Counting/filterCounts.R: has_external could be NA

### ISSUE THREAD

**Issue by schance995** [gagneurlab/drop#356]
On Drop 1.2.1 it is possible that entries in ods@colData$GENE_COUNTS_FILE is "NA" (type character) instead of NA (type NA).

```r
has_external <- !(all(ods@colData$GENE_COUNTS_FILE == "") || is.null(ods@colData$GENE_COUNTS_FILE))
```

This fix worked for me, but something more robust would be better. The issue is how NA values are read from the csv.

```r
has_external <- !(all(is.na(ods@colData$GENE_COUNTS_FILE)) || all(ods@colData$GENE_COUNTS_FILE == "") || is.null(ods@colData$GENE_COUNTS_FILE))
```

**Comment by nickhsmith** [comment]
Did you see the changes in 1.2.2 ? The sample annotation table reading is a bit more robust.

**Comment by schance995** [comment]
Yes this is fixed in 1.2.2.
